### ISSUE TITLE
X10S Express bootloader is ignoring firmware.bin on the sdcard

### ISSUE THREAD

**Issue by rafaelbiriba** [FrSkyRC/ETHOS-Feedback-Community#1811]
I just tried both 1.2.10 DFU bootloader for horus and also 1.2.11 frsk extension. 

Tried to flash the bootloader using ethos software, also opentx companion, also Dfuse software... All of them same result: Stucked on bootloader with no way to flash the firmware.

Context: I have a X10S Express with FROS and I am trying to upgrade to Ethos.
Flashing with OpenTX seens to work, so I will stick with that.

**Comment by WismyYao** [comment]
Can you see "DFU in FS mode" in device manager when you power off radio and connect the computer ?

**Comment by bsongis-frsky** [comment]
Have you followed the documentation?
Is your Ethos bootloader flashed? 
What do you see on screen?

**Comment by rafaelbiriba** [comment]
I found the problem!

I was using a `32gb SanDisk Extreme V30 UHS-I`.  During the bootloader, it actually mounts the sdcard and I could actually copy the firmware.bin into it. But after unplug, nothing was happening.

After switching to a cheap 16GB brandless card, I did the same procedure and it actually worked.

I found this by accident. After having problems flashing ethos, and tried to flash opentx (which worked), but during the boot, it was complaning about error on reading the SDCard. This was the message that makes me to believe that something is wrong with the SDCard.

Maybe would be great to add to the bootloader screen, if the sdcard is present and acessible. Somehow it mounts but can not actually read it.

**Comment by rafaelbiriba** [comment]
TL;DR:

✅ Flashed the DFU into the radio 1.2.10 version
✅ Turn on + Enter to turn on on bootloader screen
✅ Original mini usb cable connected
✅ `32gb SanDisk Extreme V30 UHS-I` mounted, so copied the sdcard content + firmware.bin
✅ Removed the cable
❌ Nothing happens.

**Comment by mawzthefinn** [comment]
Issue is most likely the use of a UHS-1 card. UHS-1 cards have a slightly different electrical interface and are not always fully backwards compatible with devices intended for older/slower cards. 

Moving to a slower Class 4, 6 or 10 card without a UHS rating is the best solution.

**Comment by rafaelbiriba** [comment]
Is there anything that could be added as feedback to the user? Even an `incompatible sdcard` error would be great!

**Comment by rafaelbiriba** [comment]
Feel free to close this issue, as I solved my problem. But my recommendation above remains! 👆🏻

**Comment by mawzthefinn** [comment]
Closing as per comment.
